ipaplatform/base/tasks.py

```diff
@@ -136,14 +136,9 @@ def modify_nsswitch_pam_stack(self, sssd, mkhomedir, statestore):
         If sssd flag is true, configure pam and nsswtich so that SSSD is used
```

Comment:
There is a typo (nsswtich instead of nsswitch) that can be fixed in your commit.